abstract.tex

```diff
@@ -18,7 +18,7 @@
 that can be run easily by anyone.
 As institutions and policies increasingly adopt Open and Reproducible policies,
 researchers urgently need practical tools to fulfill these requirements.
-Further, policy makers can benefit from tools for evaluation and enforcement.
+Further, policy makers can benefit from tools for evaluation and enforcement of reproducibility policies, once established.
```

Comment:
Why did you add "once established"? 
Do you mean that outcomes of the SOURCE project could become "policy instruments" and could encourage institutions, research councils, etc. to adopt and enforce Open and Reproducible policies?

Reply:
I think that 'once established' refers to the policies that the policymakers will establish: once they have this, Binder tools can be used to evaluate if people stick to them, or enforce them (by checking the evaluation).

But even if the above is the intended interpretation, I'd think we could drop ", once established".